seatunnel-ui/src/views/datasource/list/use-columns.ts

```diff
@@ -31,36 +31,46 @@ export function useColumns(onCallback: Function) {
       ...getTableColumn([{ key: 'id', title: t('datasource.id') }]),
       {
         title: t('datasource.datasource_name'),
-        key: 'datasourceName'
+        key: 'datasourceName',
+        align: 'center'
       },
       {
         title: t('datasource.datasource_user_name'),
-        key: 'createUserName'
+        key: 'createUserName',
+        align: 'center'
       },
       {
         title: t('datasource.datasource_type'),
         key: 'pluginName',
-        width: 180
+        width: 180,
+        align: 'center'
       },
       {
         title: t('datasource.datasource_parameter'),
         key: 'parameter',
         width: 180,
+        align: 'center',
         render: (row: any) => {
           return row.datasourceConfig
             ? h(
                 NPopover,
                 { trigger: 'click' },
                 {
                   trigger: () =>
-                    h(NButton, { text: true }, {
-                      default: () => t('datasource.click_to_view')
-                    }),
+                    h(
+                      NButton,
+                      { text: true },
+                      {
+                        default: () => t('datasource.click_to_view')
+                      }
+                    ),
                   default: () =>
                     h(JsonHighlight, {
-                      params: JSON.stringify(
-                        row.datasourceConfig
-                      ) as string
+                      params: JSON.stringify({
+                        url: row.datasourceConfig.url,
+                        driver: row.datasourceConfig.driver,
+                        user: row.datasourceConfig.user
```

Comment:
The idea to hide password makes sense to me. As datasource configurations can be different for different data source types.  can you remove only the password configuration and keeping remaining other configuration as it is, so we will not loose any extra configuration.